pkg/logs/launchers/integration/launcher_test.go

```diff
@@ -128,6 +128,39 @@ func (suite *LauncherTestSuite) TestWriteMultipleLogsToFile() {
 	assert.Equal(suite.T(), expectedContent, string(fileContents))
 }
 
+// TestEnsureFileSize tests that ensureFileSize enforces the correct file sizes for launcher files
+func (suite *LauncherTestSuite) TestEnsureFileSize() {
```

Comment:
`EnsureFileSize` is no longer a function in the code. Should we modify the test name to reflect that actual function we are testing? 

By looking at the code I think the function that ensure file size is `receiveLogs`

Reply:
Good idea, changed it to test `deleteFile`, which sort of replaced the `ensureFileSize` function.